lib/services/large_result_set.js

```diff
@@ -31,7 +31,7 @@ function LargeResultSetService(connectionConfig, httpClient)
     return err && (
       err.code === 'ECONNRESET' || err.code === 'ETIMEDOUT' ||
       // errors named with TimoutError suffix are thrown from urllib
-      err.name === 'ResponseTimeoutError' || err.name === 'ConnectionTimeoutError'
+      err.name === 'AxiosError' && err.code === 'ECONNABORTED'
```

Comment:
Can we have some parenthesis here as this is not obvious when mixing || and &&.

Reply:
it's not necessary, but we can